code/modules/hydroponics/hydroponics.dm

```diff
@@ -510,8 +510,8 @@
  */
 /obj/machinery/hydroponics/proc/mutatepest(mob/user)
 	if(pestlevel > 5)
-		message_admins("[ADMIN_LOOKUPFLW(user)] last altered a hydro tray's contents which spawned spiderlings")
-		log_game("[key_name(user)] last altered a hydro tray, which spiderlings spawned from.")
+		message_admins("[ADMIN_LOOKUPFLW(lastuser?.resolve())] last altered a hydro tray's contents which spawned spiderlings")
+		log_game("[key_name(lastuser?.resolve())] last altered a hydro tray, which spiderlings spawned from.")
```

Comment:
user should be lastuser here right? why not just let it pass down?

Reply:
`user` does work here i just thought i'd use `lastuser` because it's the variable explicitly intended for tray logs. I'll swap it over though.